examples/install-clusterwide.yml

```diff
@@ -31,22 +31,31 @@ kind: ClusterRole
 metadata:
   name: excoredns
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - services
-  - namespaces
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - extensions
-  - networking.k8s.io
-  resources:
-  - ingresses
-  verbs:
-  - list
-  - watch
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - namespaces
+    verbs:
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+      - networking.k8s.io
+    resources:
+      - ingresses
+    verbs:
+      - list
+      - watch
+  # if you need HTTPRoute, TLSRoute, GRPCoute
+  #
+  # - apiGroups:
+  #     - gateway.networking.k8s.io
+  #   resources:
+  #     - "*"
+  #   verbs:
+  #     - watch
+  #     - list
 ---
```

Comment:
why did you comment this out?

Reply:
Mainly because by default I'm not sure people test gateway apis.

Reply:
ack. sounds good.